src/components/RegistrationMessage.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import { Spinner } from "../components";
+import { Confetti } from "../assets";
+
+interface RegistrationMessageProps {
+  username: string | undefined;
+}
+
+export const RegistrationMessage = ({ username }: RegistrationMessageProps) => {
+  const [isLoading, setIsLoading] = useState<boolean>(true);
```

Comment:
No need to specify the type here, it can be inferred since the default value is `true`.